File:  blackjack-tamu-431/engine/BlackJackCard.java
package engine;

public class BlackJackCard extends Card
{
	public BlackJackCard()
	{
		super();
	}
	
	public BlackJackCard(int v, String s)
	{
		super(v,s);
	}

  	public int getValue()
	{
		int value = 0;
		
		if (super.getFace() > 10 && super.getFace() != 1)
			value = 10;
		else if (super.getFace() == 1)
			value = 11;
		else
			value = super.getFace();
		
		return value;
	}
}

File: trunk/ blackjack-tamu-431/C# version/BlackJackDealer/BlackJackDealer/BlackJackCard.cs
using System;
using System.Collections.Generic;


namespace engine
{

    public class BlackJackCard : Card
    {
        public BlackJackCard(): base()
        {
        //{
        //    //super()
        }

        public BlackJackCard(int f, string s): base(f,s)
        {

        //    //super(v,s)
        }

        new public int getValue()
        {
            int value = 0;
        
            if (base.getFace() > 10 && base.getFace() != 1)
                value = 10;
            else if (base.getFace() == 1)
                value = 11;
            else
                value = base.getFace();

            return value;
        }
    }
}

